errorprone/src/main/java/org/robolectric/errorprone/bugpatterns/RobolectricShadow.java

```diff
@@ -42,7 +42,11 @@
 import org.robolectric.annotation.Implementation;
 import org.robolectric.annotation.Implements;
 
-/** @author christianw@google.com (Christian Williams) */
+/**
+ * This setup setups Robolectric shadows
```

Comment:
What about "Ensure Robolectric shadow's method marked with {@code @Implemenetation} is protected"?

Reply:
Sure

Reply:
https://github.com/robolectric/robolectric/pull/7399/files#diff-666cc02e905ca94eb02ce55da46c9030656f881985a5fe6fe88770652c128cbcR46